data/mods/ssb/random-teams.ts

```diff
@@ -40,6 +40,12 @@ export const ssbSets: SSBSets = {
 		signatureMove: 'Data Corruption',
 		evs: {hp: 4, spa: 252, spe: 252}, nature: 'Timid',
 	},
+	Jett: {
```

Comment:
```suggestion
	Crystalites: {
```

obv move this to the correct place alphabetically, alongside the move/ability